src/vs/workbench/contrib/debug/browser/debugAccessibleView.ts

```diff
@@ -0,0 +1,103 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AccessibleViewProviderId, AccessibleViewType, IAccessibleViewContentProvider } from 'vs/platform/accessibility/browser/accessibleView';
+import { AccessibilityVerbositySettingId } from 'vs/workbench/contrib/accessibility/browser/accessibilityConfiguration';
+import { IDebugService, IReplElement } from 'vs/workbench/contrib/debug/common/debug';
+import { IAccessibleViewImplentation } from 'vs/platform/accessibility/browser/accessibleViewRegistry';
+import { ServicesAccessor } from 'vs/platform/instantiation/common/instantiation';
+import { getReplView, Repl } from 'vs/workbench/contrib/debug/browser/repl';
+import { IViewsService } from 'vs/workbench/services/views/common/viewsService';
+import { ContextKeyExpr } from 'vs/platform/contextkey/common/contextkey';
+import { Emitter, Event } from 'vs/base/common/event';
+import { Disposable } from 'vs/base/common/lifecycle';
+
+export class DebugAccessibleView extends Disposable implements IAccessibleViewImplentation {
+	priority = 70;
+	name = 'debugConsole';
+	when = ContextKeyExpr.equals('focusedView', 'workbench.panel.repl.view');
+	type: AccessibleViewType = AccessibleViewType.View;
+	constructor() {
+		super();
+	}
+	getProvider(accessor: ServicesAccessor) {
+		const viewsService = accessor.get(IViewsService);
+		const debugService = accessor.get(IDebugService);
+		return this._register(new DebugAccessibleViewProvider(viewsService, debugService));
```

Comment:
The lifecycle of a 'factory' creating a disposable object but still keeping track of it is a little odd. If `getProvider` is called multiple times it will continue to retain and 'leak' all previous created view providers. Should the consumer of the `IAccessibleViewImplentation` handle this instead?